test/nearleyc.test.js

```diff
@@ -14,46 +14,46 @@ describe("bin/nearleyc", function() {
     after(cleanup)
 
     it('builds for ES5', function() {
-        const out = externalNearleyc("grammars/parens.ne", '.js')
-        var grammar = nearley.Grammar.fromCompiled(require(`./${out}.js`));
+        const out = externalNearleyc("grammars/parens.ne", '.js');
+        const grammar = nearley.Grammar.fromCompiled(require(`./${out}.js`));
```

Comment:
Not a big deal here, but please avoid doing style changes in future PRs, without asking first; they make things harder to review. Thanks! :-)

Reply:
Yes, sorry, it was annoying me and I couldn't resist. I can remove the commit from this PR if you want.

Reply:
Nah, not a big deal! I was going to get round to it eventually... :-)